t/integration/docker-compose-redis.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.8'
+
```

Comment:
regarding this docker compose file, can we move / adjust this to docker / integration test CI actions?

Reply:
It seems like the compose is not exactly useful. I was doing some local tests with that, however I see that tox.ini already includes a bunch of containers that are suitable for integration testing. I'm about to delete the file if you don't mind

Reply:
I won't mind deleting this file